_opensearch/query-dsl/index.md

```diff
@@ -12,16 +12,49 @@ redirect_from:
 
 # Query DSL
 
-While you can use HTTP request parameters to perform simple searches, you can also use the OpenSearch query domain-specific language (DSL), which provides a wider range of search options. The query DSL uses the HTTP request body, so you can more easily customize your queries to get the exact results that you want.
+OpenSearch provides a query domain-specific language (DSL) that you can use to perform customized searches on your data.
 
-For example, the following request performs a simple search to search for a `speaker` field that has a value of `queen`.
+Query DSL provides several query categories. You can even perform more complicated searches by using different elements from each variety to find more granular data that you need.
+
+The following query DSL categories include:
+
+- **Compound boolean queries** – To search with clause rules that act as logical operators to combine several simple query terms.
+- **Term-level queries** – To search documents for one or more terms, IDs, value ranges, with optional filtering by wildcard or regex, but does not provide any relevance score or text analysis.
```

Comment:
Term-level queries calculate the relevance score, but they don't sort by relevance score. So maybe
```suggestion
- **Term-level queries** – To search documents for one or more terms, IDs, value ranges, with optional filtering by wildcard or regex. Term-level queries do not sort results by relevance score or analyze search terms.
```